suricata/update/engine.py

```diff
@@ -108,8 +108,8 @@ def load(cls, config_filename, suricata_path=None):
         conf = {}
         for line in configuration_dump.splitlines():
             try:
-                key, val = line.decode().split(" = ")
-                conf[key] = val
+                parsed_line = re.match(r'(.*?) = (.*)', line.decode())
+                conf[parsed_line.group(1)] = parsed_line.group(2)
```

Comment:
Any reason not to juse use maxsplit? https://github.com/OISF/suricata-update/pull/354/commits/39b20c37a211f0b94e693595d3312a1be65183a2